libraries/botframework-connector/src/auth/certificateServiceClientCredentialsFactory.ts

```diff
@@ -40,9 +39,9 @@ export class CertificateServiceClientCredentialsFactory extends ServiceClientCre
      *
      * @param appId Microsoft application Id related to the certificate.
      * @param x5c Value that enables application developers to achieve easy certificates roll-over in Azure AD
+     * set this parameter to send the public certificate (BEGIN CERTIFICATE) to Azure AD, so that Azure AD can use it to validate the subject name based on a trusted issuer policy.
```

Comment:
By updating the parameters doc I meant that now x5c is called certificateThumbprintOrx5c. Same in the other overloads.